lxd/instance/instance_utils.go

```diff
@@ -768,6 +768,12 @@ func CreateInternal(s *state.State, args db.InstanceArgs, clearLogDir bool) (Ins
 		args.Config["volatile.uuid"] = uuid.New()
 	}
 
+	// Generate a new `volatile.uuid.generation`
+	// to differentiate this backed up instance from the original.
+	if args.Config["volatile.uuid.generation"] == "" {
```

Comment:
As this is in CreateInternal, is there a particular reason for this logic, rather than just:

```go
args.Config["volatile.uuid.generation"] = args.Config["volatile.uuid"]
```

Reply:
Also the comment doesn't make sense in all contexts of CreateInternal I'm afraid, so best not have it.